orttraining/pytorch_frontend_examples/nvidia-bert/LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
Would it be better to replace this file with a pointer to the LICENSE file in the original repo?

Reply:
I am not sure whether this is allowed. Apache license allows anyone performs any change to/distribution of the original software as long as the original license and copyright are "included". Maybe "pointer" is not considered "inclusion" of license.

If any change is performed to the original source code, we also have to list them